charger/keba-modbus.go

```diff
@@ -111,6 +112,8 @@ func NewKebaFromConfig(ctx context.Context, other map[string]interface{}) (api.C
 	var hasEnergyMeter bool
 	var hasRFID bool
 
+	wb.energyFactor = 1e4
```

Comment:
Der Default gehört m.E. in `NewKeba` denn sonst funktioniert das Ergebnis daraus gar nicht.

Reply:
Analog regEnable

Reply:
Moved.